### PR TITLE
strelaypoolsrv: remove google map and add leaflet map

### PR DESCRIPTION
**Pull request by keevBush**
Remove google map and add Leaflet for relays.syncthing.net

### CODE REVIEW

File: cmd/strelaypoolsrv/gui/index.html [hunk 1]
```diff
@@ -9,8 +9,14 @@
     <meta name="author" content=""/>
 
     <title>Relay stats</title>
-    <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet"/>
-    <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.13/css/all.css"/>
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet"/>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css"/>
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
```
Comment (AudriusButkevicius) [inline cmd/strelaypoolsrv/gui/index.html]: You should probably keep the hrefs without protocol, aka `//....`, then you don't have browser errors when accessing the page via http oppose to https, etc.

File: cmd/strelaypoolsrv/gui/index.html [hunk 2]
```diff
@@ -38,11 +44,11 @@
     <div class="container">
       <h1>Relay Pool Data</h1>
       <div ng-if="relays === undefined" class="text-center">
-        <img src="//cdnjs.cloudflare.com/ajax/libs/galleriffic/2.0.1/css/loader.gif" alt=""/>
+        <img src="https://cdnjs.cloudflare.com/ajax/libs/galleriffic/2.0.1/css/loader.gif" alt=""/>
         <p>Please wait while we gather data</p>
       </div>
       <div>
-        <div ng-show="relays !== undefined" class="ng-hide">
+        <div ng-show="relays === undefined" class="ng-hide">
```
Comment (calmh) [inline cmd/strelaypoolsrv/gui/index.html]: pretty sure this change is incorrect

File: cmd/strelaypoolsrv/gui/index.html [hunk 3]
```diff
@@ -185,9 +191,8 @@ <h1>Relay Pool Data</h1>
 
 
     <script type="text/javascript" src="//code.jquery.com/jquery-2.1.4.min.js"></script>
-    <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.8/angular.min.js"></script>
+    <script type="text/javascript" src="http://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.8/angular.min.js"></script>
```
Comment (calmh) [inline cmd/strelaypoolsrv/gui/index.html]: no `http`, either `//...` as it was to keep whatever the browser is using, or `https` if we need to hardcode something for some reason

File: cmd/strelaypoolsrv/gui/index.html [hunk 4]
```diff
@@ -228,11 +233,13 @@ <h1>Relay Pool Data</h1>
...
-      mapTypeId: google.maps.MapTypeId.ROADMAP
-    });
-    $scope.mapBounds = new google.maps.LatLngBounds();
+    $scope.map = L.map('map').setView([40.90296, 1.90925], 2);
+    L.tileLayer('http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+    {
+      attribution: 'Leaflet',
+      maxZoom: 17
+    }).addTo($scope.map);
+    // $scope.mapBounds = new google.maps.LatLngBounds();
```
Comment (calmh) [inline cmd/strelaypoolsrv/gui/index.html]: remove

File: cmd/strelaypoolsrv/gui/index.html (hunk 2 above)
Comment (keevBush) [reply cmd/strelaypoolsrv/gui/index.html]: Yes it me I change that when I load the map

File: cmd/strelaypoolsrv/gui/index.html [hunk 5]
```diff
@@ -38,7 +45,7 @@
     <div class="container">
       <h1>Relay Pool Data</h1>
       <div ng-if="relays === undefined" class="text-center">
-        <img src="//cdnjs.cloudflare.com/ajax/libs/galleriffic/2.0.1/css/loader.gif" alt=""/>
+        <img src="https://cdnjs.cloudflare.com/ajax/libs/galleriffic/2.0.1/css/loader.gif" alt=""/>
```
Comment (AudriusButkevicius) [inline cmd/strelaypoolsrv/gui/index.html]: needs https removed

Comment (calmh) [reply cmd/strelaypoolsrv/gui/index.html]: Does it though? I think the root cause here is that it's difficult to test these pages without a fully fledged strelaypoolserv/ursrv setup running, so they're opened as files and then the `//...` url's don't work. I see no harm in hardcoding https for all of these

Comment (keevBush) [reply cmd/strelaypoolsrv/gui/index.html]: Yes I have the same problem. For resolve there I had https for all of there

**Comment by calmh** [comment]
In principle lgtm, but there is some weirdness with the circles. They are not centered on the markers:

Screen Shot 2020-03-28 at 11 47 52
Screen Shot 2020-03-28 at 11 48 07

**Comment by AudriusButkevicius** [comment]
While at it, we might want to change the circles to log scale.

**Comment by AudriusButkevicius** [comment]
Also, the ng-mouseover stuff that puts the relay to focus when you hover over it in the list below probably does not work anymore.

**Comment by AudriusButkevicius** [comment]
There is also stuff like
```
      $scope.map.fitBounds($scope.mapBounds);
      if ($scope.relays.length == 1) {
        $scope.map.setZoom(13);
      }
```

which I am not sure if it's supported in the new map thing?

**Comment by AudriusButkevicius** [comment]
I don't even see $scope.mapBounds existing anymore.

**Comment by keevBush** [comment]
I look that problem is leaflet. More youb zoom more precision on the map is revelated. Is Leaflet

**Comment by keevBush** [comment]
Pin update his precision when You zoom

**Comment by AudriusButkevicius** [comment]
There is still the missing functionality of the relay being "moused over and focused" when you hover over the row for the relay in the data table below.

**Comment by AudriusButkevicius** [comment]
It seems the circles are in the wrong place after the adjustments that prevent stuff from overlapping:
image

**Pull request merged**